api/src/nlp/controllers/nlp-value.controller.spec.ts

```diff
@@ -125,7 +125,7 @@ describe('NlpValueController', () => {
         (acc, curr) => {
           const ValueWithEntities = {
             ...curr,
-            entity: nlpEntities[parseInt(curr.entity)].id,
+            entity: nlpEntities[parseInt(curr.entity!)].id,
```

Comment:
```suggestion
            entity: curr.entity ? nlpEntities[parseInt(curr.entity!)].id : null,
```
Maybe we should be careful about this ?

Reply:
Yes 100%